static/styles/zulip.css

```diff
@@ -1462,6 +1454,17 @@ td.pointer {
     opacity: 1;
 }
 
+.message_header + .selected_message {
+    /* Sticky message header overlaps with the box-shadow, so we add another box-shadow to
+       compensate for that. */
+    .messagebox-content {
+        box-shadow: inset 0 1px 0 2px hsl(215, 47%, 50%),
+            inset 0 0 0 2px hsl(215, 47%, 50%), -1px -1px 0 0 hsl(215, 47%, 50%),
+            1px 1px 0 0 hsl(215, 47%, 50%), -1px 1px 0 0 hsl(215, 47%, 50%),
+            1px -1px 0 0 hsl(215, 47%, 50%);
```

Comment:
One would think it shouldn’t take six shadows to draw a rectangle. We can at least replace the last four non-`inset` shadows (`-1px -1px 0 0`, `1px 1px 0 0`, `-1px 1px 0 0`, `1px -1px 0 0`) with one (`0 0 0 1px`); this applies to the original rule too.

Reply:
(And, just noting, we’ll need to remember to remove this in #23531.)

Reply:
Yeah, updated!

Reply:
Where is this code to be removed (wasn't sure because this comment is on an outdated diff)? 

Working on #23531 right now :)

Reply:
It is in the commit [css: Combine box-shadows for selected message.](https://github.com/zulip/zulip/pull/23900/commits/82229b5da216b83c7fc3ea11a909c2a22a3b5512)

Reply:
Ah okay great :) So I've already addressed it. Thanks!